receiver/vcenterreceiver/testdata/metrics/expected-all-enabled.yaml

```diff
@@ -2924,14 +2924,14 @@ resourceMetrics:
           - description: The congestions of IOs generated by all vSAN clients in the host.
             gauge:
               dataPoints:
-                - asInt: "1"
```

Comment:
these are only 0 because you're converting to int. you could pick larger values in your mocked responses so they nicely divide by the interval (300) to get whole number results here, but not necessary

Reply:
resolve